devops/kubernetes/argus/Pulumi.yaml

```
@@ -0,0 +1,30 @@
+name: argus
+runtime: nodejs
+description: A Pulumi program to deploy Argus node
+template:
+  config:
+    aws:profile:
+      default: joystream-user
+    aws:region:
+      default: us-east-1
+    isLoadBalancerReady:
+      description: Whether the load balancer service is ready and has been assigned an IP
+      default: false
+    queryNodeHost:
+      description: Query node GraphQL endpoint
+      default: 'https://hydra.joystream.org/graphql'
+    wsProviderEndpointURI:
+      description: Chain RPC endpoint
+      default: 'wss://rome-rpc-endpoint.joystream.org:9944/'
+    argusImage:
+      description: The distributor node image to use for running the node
+      default: joystream/distributor-node:latest
+    keys:
+      description: Specifies the keys available within distributor node CLI
+      default: "[{ 'suri': '//Alice' }]"
```

```suggestion
      default: "[{ \"suri\": \"//Alice\" }]"
```

Escape `"` to make valid JSON.

As a general point, When is this default value even used? If we create a new stack with `pulumi stack init` we just get a blank with no default config values, so its not like Pulumi.yaml is used as a "template". 

And if there is a default value set in this yaml file, why do we also do `const keys = config.get('keys') || '[{ "suri": "//Alice" }]'` in index.ts effectively overriding it.

---

The default value in `Pulumi.yaml` is more of a reference guide for a user to see what is the default value for a parameter so that they don't have to dive into the code.